gridBuilder.umd.js

```diff
@@ -52,6 +52,7 @@
 			this.html = this.createGrid($(this.el), config);
 			var css = this.createStyles(config);
 			$('#grid-css').html(css);
+			showWarnings();
```

Comment:
showWarnings shouldn't be tightly coupled with gridBuilder. I think you can use statechanged event at the app level.